src/Controller/ImageController.php

```diff
@@ -36,8 +36,8 @@ public function image(string $filename, Request $request): StreamedResponse
             'cache' => $this->config->getPath('cache', true, 'thumbnails'),
         ]);
 
-        if ($path = $request->query->get('path')) {
-            $filename = sprintf('%s/%s', $path, $filename);
+        if ($request->query->get('path')) {
```

Comment:
query->has('path')

Reply:
Fixed, and in another location as well..